### ISSUE TITLE
44.1k vae training details

### ISSUE THREAD

**Issue by CrossLee1** [hkchengrex/MMAudio#69]
hi
I'm using Make-An-Audio-2 repo to reproduce the training for 44.1k VAE, but got results not as good as the released model.

could you provide the details for training?

1. how many GPUs used
2. how many steps used
3. if possible, could you please provide the training curves
Very thanks~~

**Comment by hkchengrex** [comment]
The main change is the normalized network architecture, but as noted in the paper, that did not lead to any noticeable empirical difference in generation.

For the other parameters, we simply choose a set that results in fast training and convergence (the original batch size is too small, and there are too many iterations) without much tuning.
- Batch size: 512
- GPUs: 8
- # iterations: 300K
- Learning rate: 2e-3 (for both the generator and the discriminator)
- disc_start: 20001
